crux-bench/src/crux/bench/main.clj

```diff
@@ -1,25 +1,34 @@
 (ns crux.bench.main
   (:require [clojure.string :as string]
             [crux.bench :as bench]
+            [crux.bench.sorted-maps-microbench :as sorted-maps]
             [crux.bench.ts-devices :as devices]
             [crux.bench.ts-weather :as weather]
             [crux.bench.watdiv-crux :as watdiv-crux]))
 
+(defn post-to-slack [results]
+  (doto results
+    (-> (bench/results->slack-message) (bench/post-to-slack))))
+
 (defn -main []
   (bench/post-to-slack (format "*Starting Benchmark*, Crux Version: %s, Commit Hash: %s\n"
                                bench/crux-version bench/commit-hash))
 
-  (let [bench-results (bench/with-nodes [node]
-                        [(doto (devices/run-devices-bench node)
-                           (bench/post-slack-results :ts-devices))
+  (let [bench-results (concat (bench/with-nodes [node (select-keys bench/nodes ["standalone-rocksdb"])]
```

Comment:
Think this should be  `(select-keys bench/nodes ["standalone-rocksdb", "kafka-rocksdb""])`.

Reply:
By default, makes sense (to me) to run it on all of the nodes (as we do on our nightly bench).

Reply:
yep - didn't need it initially but will do shortly, will add